exporters/tests/data/webhook_pelorus_deploytime.json

```diff
@@ -1,6 +1,5 @@
 {
   "app": "mongo-todolist",
   "image_sha": "sha256:af4092ccbfa99a3ec1ea93058fe39b8ddfd8db1c7a18081db397c50a0b8ec77d",
-  "namespace": "mongo-persistent",
-  "timestamp": 1557933657
+  "namespace": "mongo-persistent"
```

Comment:
ok to change for deploytime, but others test data should not be touched

Also this breaks `exporters/webhook/README.md` steps, fix it or remove it from there

Reply:
Ok updated the README and added back timestamp as it's a valid to have it there to present how it should be used.